src/many/src/cose_helpers.rs

```diff
@@ -12,10 +12,16 @@ use std::collections::{BTreeMap, BTreeSet};
 /// * `x` - Public key
 /// * `d` - Private key
 pub fn eddsa_cose_key(x: Vec<u8>, d: Option<Vec<u8>>) -> CoseKey {
-    let mut params: Vec<(Label, Value)> = Vec::from([(
-        Label::Int(coset::iana::OkpKeyParameter::X as i64),
-        Value::Bytes(x),
-    )]);
+    let mut params: Vec<(Label, Value)> = Vec::from([
```

Comment:
I think we should move that change in a separate PR ASAP so people will get their new ID quick.

Reply:
Can do.

Reply:
See #86